composer.json

```diff
@@ -27,7 +27,7 @@
         "justinrainbow/json-schema": "^3.0 || ^4.0 || ^5.0",
         "mlocati/composer-patcher": "^1.2.2",
         "padraic/phar-updater": "~1.0@dev",
-        "symfony/console": "^5.0"
+        "symfony/console": "^5.3.7"
```

Comment:
Why did you change this line? Does this project really needs at least version 5.3.7?

Reply:
Yes, because PHP 8.1 support is added from that point.

Reply:
> because PHP 8.1 support is added from that point.

That's correct. But it's enough to have 5.3.7+ in composer.lock.
Having ^5.3.7 in composer.json forces other projects that include pickle to have this requirement.
If pickle doesn't use features included in 5.3.7+ features, it's enough to have ^5.0

Reply:
Actually, I think the tests should test `lowest` and `highest` dependencies and not rely on `composer.lock`.

Reply:
Also, the docs only mention to install pickle with the phar...

Reply:
So it could even become Symfony 5.4

Reply:
> the docs only mention to install pickle with the phar

Sure, but since pickle is published on packagist, people may use its features in their projects too.